util/sdks/go.sh

```diff
@@ -66,10 +66,11 @@ build_go_sdk() {
     if [[ -d "$DIR_PATCHED_PROXY_WASM_GO_SDK" \
           && -f "$DIR_PATCHED_PROXY_WASM_GO_SDK/.hash" \
           && $(cat "$DIR_PATCHED_PROXY_WASM_GO_SDK/.hash") == $(echo $hash_src)
-          && -z "$clean" ]];
+          && -z "$clean" ]] && \
+       find $DIR_PATCHED_PROXY_WASM_GO_SDK -name '*.wasm' | grep -q .
     then
         notice "Go examples already built"
-        exit
+        return
```

Comment:
Can this stay `exit` like `assemblyscript.sh`, or the later be switch to also `return`? Aiming at keeping both scripts identical.

Reply:
Both should be `return`. It was a mistake on my part.

Reply:
Ok I see now, they really need to be `return` indeed. Another question for you:

```sh
make build.examples || exit 0
```

Does this look right? Should we silently fail if `make build.examples` errors-out? 🤔

Reply:
> Does this look right to you? Should we silently fail if make build.examples errors-out?

I was tempted to change it to `exit 1` and even did it for a bit, but backtracked last minute because I didn't really chase the history to see how that ended up there, and given the explicit `0` there might have been a reason I don't know/remember, so in the end I wanted to keep only the necessary fix in this PR and not go after other unrelated improvements.